README.md

```diff
@@ -78,8 +76,9 @@ channels:
 
 ### TOC Liaisons
 
-- Emily Fox ([@TheFoxAtWork](https://github.com/TheFoxAtWork)), Apple
-- Dave Zolotusky ([@dzolotusky](https://github.com/dzolotusky)), Spotify
+- Matt Farina ([@mattfarina](https://github.com/mattfarina)), SUSE
```

Comment:
We were just assigned new TOC Liaisons in the last week or so. Emily and Dave are our new liaisons, so can you revert this change?